packages/discovery-react-components/src/components/CIDocument/utils/__tests__/elementFromPoint.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  elementFromPoint,
+  elementFromPointMs,
+  elementFromPointFallback,
+  MsDocument
+} from '../elementFromPoint';
+
+const clientX = 50;
+const clientY = 50;
+const className = 'field--rect';
+const stopElem = null;
+
+const classList = (list: string[]): any => {
+  return {
+    contains: (name: string): boolean => {
+      return list.includes(name);
+    }
+  };
+};
+
+const mockElements = [
+  { classList: classList(['mockName']), style: { pointerEvents: null } },
+  { classList: classList(['mockClass']), style: { pointerEvents: null } },
+  { classList: classList([className]), style: { pointerEvents: null } }
```

Comment:
Reading this again, I think I would prefer if there were an actual string here instead of the `className` variable. Then in each test, we would use same string. Makes it more readable.